Automattic/woocommerce-payments
# chore: copy prb blocks implementation to tokenized cart

Part of https://github.com/Automattic/woocommerce-payments/issues/8527

#### Changes proposed in this Pull Request









Discussed in team meeting 2024-06-19.
To make things easier to review, I am copying the PRB blocks implementation to the tokenized cart directory, without changing files/behavior.
We want to keep the two implementations separate for an easier time with cleanup.
All files have been copied over and will be tweaked for the final implementation.

#### Testing instructions





- Enable the "Enable Cart-Token implementation for PRBs" flag in your dev tools (you'll need to update them)
- Ensure you have Google Pay/Apple Pay enabled in the merchant's settings
- Add a product to the cart
- To to the block-based checkout page
- There should be a Google Pay/Apple Pay button and an element called `express-payment-method-woocommerce_payments_tokenized_cart_payment_request` on the page (with the tokenized cart feature flag disabled, the element is instead called `express-payment-method-woocommerce_payments_payment_request`)

-------------------


**Post merge**
